examples/test/emitter.cc

```diff
@@ -62,17 +60,15 @@ int main(int argc, char* argv[])
         msg->setAddress(1001);
 
         std::memcpy(frame + sizeof(EthercatHeader), msg->data(), msg->size());
-        header->len = msg->size();
+        header->len = static_cast<uint16_t>(msg->size() & 2047); // Ensures that size() won't use more than 11 bits
```

Comment:
2047 -> prefer hexadecimal representation. Not sure if the static_cast is needed with the mask.